### DISCUSSION TITLE
Why number of voices on app is lesser than on browser?

### DISCUSSION THREAD

**Discussion by LeitHunt** [jing332/tts-server-android#185] [General]
I love one voice [MultiLingal Brian] which is available on browser but not on App, why?
Thanks for such a great app :)

**Comment by setzke** [comment]
Shows up for me. Add built-in, change language to English (US) and he's in the list. Pixel 8.

![Screenshot_20240504-102231](https://github.com/jing332/tts-server-android/assets/13110348/20a18060-0d95-4afe-99ce-9bbba9254f8d)

**Reply by LeitHunt** [reply]
![photo_2024-05-11_12-10-28](https://github.com/jing332/tts-server-android/assets/28060751/df1b6da8-a057-4b90-82ad-2572f9168863)
Strange in my case it's not appearing. Only options visible in the screenshot are visible. Pixel 6A
